packages/uicore/src/components/MultiTypeInput/LearnMorePopover.tsx

```diff
@@ -37,7 +37,8 @@ const helperText: Record<MultiTypeInputType, React.ReactNode> = {
   ),
   [MultiTypeInputType.EXECUTION_TIME]: (
     <React.Fragment>
-      <b>Runtime Inputs</b> are placeholders for values that will be provided during a Pipeline execution.
+      <b>Execution Inputs</b> are placeholders for values that must be provided when manually initiating a pipeline
+      execution, when triggering it, and when certain steps require inputs when the pipeline is running.
```

Comment:
strings ?

Reply:
These can be defined inline as strings won't be accessible in uicore